lib/sparql_to_sw_solr/instance_solr_doc.rb

```diff
@@ -75,6 +75,13 @@ def solution_values_for_binding(solutions, binding_symbol)
       end
     end
 
+    def concatenate_values(val1, separator, val2)
+      result = "#{val1}" \
+      "#{present?(val1) && present?(val2) ? separator : nil}" \
+      "#{val2}"
+      result unless result.empty?
+    end
```

Comment:
thanks for the tip.  I gave it a shot, and 33 specs failed -- it seems to add the separator even when the values are nil.